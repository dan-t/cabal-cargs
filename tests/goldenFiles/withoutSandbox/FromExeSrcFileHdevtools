-g-iexe -g-W -g-optP-DEXE -g -g-idist/build/autogen -g-Idist/build/autogen -g-optP-include -g-optPcabal_macros.h --socket=.hdevtools.sock