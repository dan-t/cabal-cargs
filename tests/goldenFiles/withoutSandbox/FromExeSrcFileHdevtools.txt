-g-iexe -g-W -g-optP-DEXE -g-idist/build/autogen -g-Idist/build/autogen -g-optP-include -g-optPcabal_macros.h --socket=.hdevtools.sock